uqcsbot/scripts/wiki.py

```diff
@@ -0,0 +1,26 @@
+from uqcsbot import bot, Command
+import requests
+import json
+
+
+@bot.on_command("wiki")
+async def handle_wiki(command: Command):
+    search = command.arg
+    url = f"https://en.wikipedia.org/w/api.php?action=opensearch&search={search}&format=json"
+    http_response = await bot.run_async(requests.get, url)
+    response = json.loads(http_response.content)
+
+    # If the results are empty let them know
+    try:
+        info = response[2][0]
```

Comment:
what's the 2 and 0 here mean? give some description :)

also, it'd be better to do this in an if statement to remove an unnecessary try/except